codalab/lib/bundle_cli.py

```diff
@@ -2758,34 +2759,42 @@ def do_uinfo_command(self, args):
             user = client.fetch('user')
         else:
             user = client.fetch('users', args.user_spec)
-        self.print_user_info(user)
+        self.print_user_info(user, args.field)
+
+    def print_user_info(self, user, fields):
+        def print_attribute(key, user):
+            # These fields will not be returned by the server if the
+            # authenticated user is not root, so don't crash if you can't read them
+            if key in ('last_login', 'email', 'time', 'disk'):
+                try:
+                    if key == 'time':
+                        value = formatting.ratio_str(
+                            formatting.duration_str,
+                            user['time_used'],
+                            user['time_quota'])
+                    elif key == 'disk':
+                        value = formatting.ratio_str(
+                            formatting.size_str,
+                            user['disk_used'],
+                            user['disk_quota'])
+                    else:
+                        value = user[key]
+                except KeyError:
+                    pass
+            else:
+                value = user.get(key, None)
 
-    def print_user_info(self, user):
-        def print_attribute(key, value):
             print >>self.stdout, u'{:<15}: {}'.format(key, value).encode('utf-8')
 
-        for key in ('id', 'user_name', 'first_name', 'last_name',
-                    'affiliation', 'url', 'date_joined'):
-            print_attribute(key, user.get(key, None))
+        default_fields = ('id', 'user_name', 'first_name', 'last_name',
+                    'affiliation', 'url', 'date_joined', 'last_login',
+                    'email', 'time', 'disk')
+
+        fields = fields.split(',') if fields else default_fields
+
+        for field in fields:
+            print_attribute(field, user)
```

Comment:
There should be two modes: one is pretty printing all the attributes, and the other is printing out individual fields (which should not have the field names).  Just match the behavior of `cl info -f`.